src/app/portlets/dot-browser/dot-browser-component.ts

```diff
@@ -1,14 +1,82 @@
-import {Component, OnInit, ViewEncapsulation} from '@angular/core';
```

Comment:
If you have the time I recommend to pass prettier in your files: https://github.com/prettier/prettier/blob/master/editors/webstorm/README.md